src/IssueCreator/Logging/FileLogger.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IssueCreator.Logging
+{
+    public class FileLogger
+    {
+        private string _logPath;
+        public FileLogger(string logPath = "")
+        {
+            if (string.IsNullOrEmpty(logPath))
+            {
+                logPath = "log.txt";
```

Comment:
But where? CWD could be anything. Might default to your app's ClickOnce data directory (I forget how to get it off hand - it's been years since I wrote/modified one.)

Reply:
This is the default -- and you can get to this folder if you really want to, but this is specified as an input parameter. I will throw if none is specified.